Now checks siad status on each errored API call

Mingling94 opened the pull request:
The logic being that if a call errored, '/consensus' better still be
reachable or siad is definitely not running anymore

Fixes #53

cyrozap commented:
Source: https://stackoverflow.com/questions/21634349/heroku-npm-postinstall-script-to-run-grunt-task-depending-on-enviro

cyrozap commented:
Test comment, please ignore

seveibar commented:
bravo, npm is awesome

zlalvani commented:
I pulled from Forest and it's working.

cyrozap commented:
Did you test this after uninstalling grunt globally? i.e., did you do a `npm uninstall -g grunt` before testing? I want to make absolutely sure this still works after doing that. If it works, feel free to merge this PR.

zlalvani commented:
I actually did this on a different Windows install, so I ran npm install to grab the needed dependencies.

Mingling94 committed:
Now checks siad status on each errored API call

The logic being that if a call errored, '/consensus' better still be
reachable or siad is definitely not running anymore
